geonode/templates/_permissions_form.html

```diff
@@ -341,7 +341,7 @@ <h4 class="modal-title" id="exampleModalLabel">{% trans "Message box" %}</h4>
 
     /* OpenLayers */
     {% if resource.ll_bbox %}
-      var bbox = [{{ resource.ll_bbox_string }}];
+      var bbox = [{{ resource.bbox_string }}];
```

Comment:
The bbox here should always be returned into 4326 SRS. Please, revert to ll_bbox_string.